op-node/rollup/derive/plasma_data_source.go

```diff
@@ -67,12 +67,6 @@ func (s *PlasmaDataSource) Next(ctx context.Context) (eth.Data, error) {
 			s.log.Warn("invalid commitment", "commitment", data, "err", err)
 			return nil, NotEnoughData
 		}
-		// only support keccak256 commitments for now.
```

Comment:
Does this break the unit tests? I see this breaking the CI tests, but for the moment, it's probably better to still feature flag the generic commitment type

Reply:
nope it was just breaking devnet tests when using the `DA_SERVICE=true` 😄 
out of curiosity how do you see this breaking CI tests?

Reply:
sorry, that's what I meant by CI